frontend/talentsearch/resources/js/components/skills/SkillAccordion/SkillAccordion.tsx

```diff
@@ -127,8 +127,7 @@ const SkillAccordion: React.FunctionComponent<SkillAccordionProps> = ({
       <div>
         <p>
           <span data-h2-font-color="b(lightpurple)" title="award">
-            {" "}
-            {title}{" "}
+            {title}
```

Comment:
You should keep that empty space for formatting as DefaultMessage doesn't seem to preserve that needed space
https://61e099a1bb1465003a73d3ce-cjoitwqyuy.chromatic.com/?path=/story/skill-accordion--accordion-award-example